public/javascripts/needleeditor.js

```diff
@@ -171,6 +171,12 @@ NeedleEditor.prototype.LoadBackground = function(url) {
   }
 }
 
+NeedleEditor.prototype.LoadTags = function(tags) {
+  var editor = this;
+  editor.needle['tags'] = tags;
+  this.UpdateTextArea();
+}
```

Comment:
Why would you assign editor for one call and then use this the 2nd line? Why not use this in both lines?